drivers/pinmux/stm32/pinmux_stm32l4x.h

```diff
@@ -133,7 +133,7 @@
 #define STM32L4X_PINMUX_FUNC_PC6_USART6_TX  \
 	(STM32_PINMUX_ALT_FUNC_8 | STM32_PUSHPULL_PULLUP)
 #define STM32L4X_PINMUX_FUNC_PC7_USART6_RX  \
-	(STM32_PINMUX_ALT_FUNC_8 | STM32_PUPDR_NO_PUL
+	(STM32_PINMUX_ALT_FUNC_8 | STM32_PUPDR_NO_PULL)
```

Comment:
I wonder how it is possible this hasn't been seen before.
Can you push this as a separate bug fix commit so we can get it in 1.14?

Reply:
PR #14054